tests/Oryx.BuildImage.Tests/DotNetCore/DotNetCoreSampleAppsTest.cs

```diff
@@ -361,7 +369,7 @@ public void Builds_Net6BlazorWasmApp_RunsAOTCompilationInstallCommands()
                 .AddDefaultTestEnvironmentVariables()
                 .AddBuildCommand(
                 $"{appDir} --platform dotnet " +
-                $"--platform-version 6")
+                $"--platform-version {DotNetCoreRunTimeVersions.NetCoreApp60}")
```

Comment:
I made this change so that hopefully in the future we do not run into the issue where adding to the dev env will break the validation tests for all other runs, since it was pulling the most recent version of 6 but trying to use the hardcoded `DotNetCoreSdkVersions.DotNet60SdkVersion` for [this test](https://github.com/microsoft/Oryx/pull/1436/files#diff-57cf456c076f2617529dd577fc8fe89b6d7ecaf9c925b61ceb136516dae07e9fR390) 

See [this build job](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=6311951&view=logs&j=cf1ac521-3c67-5038-03af-743589bde38f&t=95ebba3e-eb3a-56d5-2c17-f0cfd59b5222&s=ff05ad62-bb9a-53b6-ce9f-72f329a63e7c) for an example failure that happened when I put the 6.0.301 sdk on dev.